README.md

```diff
@@ -97,10 +97,11 @@ You can reference them inside queries, allowing users to configure parameters su
 
 You can use the following macros in your queries
 
-| Macro Name | Syntax                     | Description                                                          | Example                                                                              |
-| ---------- | -------------------------- | -------------------------------------------------------------------- | ------------------------------------------------------------------------------------ |
-| multiVar   | `$__multiVar(prefix,$var)` | Expands a multi value variable into github query string              | `$__multiVar(label,$labels)` will expand into `label:first-label label:second-label` |
-|            |                            | When using **all** in multi variable, use **\*** as custom all value |                                                                                      |
+| Macro Name | Syntax                     | Description                                                              | Example                                                                              |
+| ---------- | -------------------------- | ------------------------------------------------------------------------ | ------------------------------------------------------------------------------------ |
+| multiVar   | `$__multiVar(prefix,$var)` | Expands a multi value variable into github query string                  | `$__multiVar(label,$labels)` will expand into `label:first-label label:second-label` |
+| day        | `$__toDay(diff)`           | Returns the day according to UTC time, a difference in days can be added | `created:$__toDay(-7)` on 2022-01-17 will expand into `created:2022-01-10`           |
+|            |                            | When using **all** in multi variable, use **\*** as custom all value     |                                                                                      |
```

Comment:
Seems like this line should go before the `day` description, as it's related to multi variable.